src/components/Cards/CardLarge/CardLarge.scss

```diff
@@ -0,0 +1,54 @@
+@import 'styles/utilities/colors.scss';
+@import 'styles/fonts.scss';
+@import 'styles/utilities/typography.scss';
+
+$card-image-height: 330px;
```

Comment:
Could you sync with design on this, the image should change sizes on diff screens IMO